src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/VideoLinkBookingMigrationService.kt

```diff
@@ -57,11 +57,11 @@ class VideoLinkBookingMigrationService(
       videoLinkBooking.prisonId = nomisMainAppointment.agencyId
       videoLinkBooking.comment = nomisMainAppointment.comment
 
-      if (preAppointment != null && preAppointment?.let { updateAppointment(it) } == null) {
+      if (preAppointment != null && updateAppointment(preAppointment) == null) {
         videoLinkBooking.appointments.remove(HearingType.PRE)
         outcomes.add(PRE_DELETED)
       }
-      if (postAppointment != null && postAppointment?.let { updateAppointment(it) } == null) {
```

Comment:
Do we def not need `preAppointment?.let` and `postAppointment?.let` anymore? Appriciate it looks like we check the same thing twice with not equal to null but maybe this is checking for another falsely value like undefined?

Reply:
This is a small code improvement not related to actual code cleaning.   
postAppointment?.let { updateAppointment(it) } execute updateAppointment when postAppointment is not null however we added postAppointment != null what means in case postAppointment is null we never reach this point postAppointment?.let { updateAppointment(it) } it means we do not need to do this check again.